src/mono/wasm/runtime/invoke-cs.ts

```diff
@@ -255,7 +260,6 @@ type BindingClosure = {
 }
 
 export function invoke_method_and_handle_exception(method: MonoMethod, args: JSMarshalerArguments): void {
-    assert_bindings();
     const fail_root = mono_wasm_new_root<MonoString>();
```

Comment:
is it intentional that you removed assert_bindings here and didn't add anything to replace it?

Reply:
it's a mistake, thanks!